.github/dependabot.yml

```diff
@@ -27,3 +27,14 @@ updates:
       - "team"
       - "team: infra"
       - "waiting for tree to go green"
+  - package-ecosystem: "github-actions"
+    directory: "/"
```

Comment:
This is supposed to be configured in /?

Reply:
Yes, this is the recommendation from dependabot docs: https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot